src/helpers/mdObjBucketHelper.ts

```diff
@@ -27,3 +27,14 @@ export function getSecondaryMeasuresLocalIdentifiers(buckets: VisualizationObjec
         get(item, "measure.localIdentifier"),
     );
 }
+
+export function getMeasuresFromMdObject(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
```

Comment:
not sure if this is right file. Look around, all functions consume as a first param "buckets". It is reason why this is called mdObjectBUCKETHelper

Reply:
or change the input param from mdObject to its buckets and change call conventions where used